docs/search/search-query-language/search-operators/macro.md

```diff
@@ -0,0 +1,55 @@
+---
+id: macro
+title: macro Operator
+sidebar_label: macro
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+With `macro` operator you can reuse the portions of search query language in multiple other queries. This operator also includes arguments and also performs its respective evaluation of the arguments.
+
+Use backquote (``) character to refer macro operator in the query. Macros referred inside another macro is called **Nested macros**. Macro referred within the context of another macro query of is called **Inner Macro**. The macro where the inner macro is referred is called the **Outer Macro**.
+
+:::note
+- Only **Administrators** and **Users** with access to **Query Reference** will be able to run queries using macros.
+- Only users with **Administrator** access can create or delete the macro. 
```

Comment:
```suggestion
- Only users with **Administrator** access can create or delete macros.
```

Reply:
We should remove `delete` here it is not a feature we support unless customer asks for it